addons/website_hr_recruitment/controllers/main.py

```diff
@@ -116,6 +116,18 @@ def jobs_thankyou(self, **post):
         # Retro-compatibility for saas-3. "phone" field should be replace by "partner_phone" in the template in trunk.
         value['partner_phone'] = post.pop('phone', False)
 
+        is_partner_list = request.session.model('res.partner').search([('email', '=', post.get('email_from').replace(" ", ""))], 0, 0, 0, request.session.context)
```

Comment:
this will break if `post.get('email_from')` is `False`... I would simply search with an `ilike` operator